### PR TITLE
Debrand automation

### PR DESCRIPTION
**Pull request by jeffkamo**
In an effort to get a new generated PWA project to the right starting spot, I've introduced a way to ensure that the project styles can be set to appear like the UI Kit at generation time. The way it works is like this:

```scss
// Scaffold (BEFORE generating)
.my-whatever-rule {
    //                         ↓             ↓  colon and semi-colon are REQUIRED!
    color: red; // ui_kit_value: $brand-color;
    border: 2px solid blue; // ui_kit_value: 1px solid $secondary-color; // 1) This comment is preserved
}

// Generated Project Output
.my-whatever-rule {
    color: $brand-color;
    border: 1px solid $secondary-color; // 1) This comment is preserved
}
```

> _Notice that the ui kit value starts with a colon and ends with a semi-colon. This is **REQUIRED!**_

 **JIRA**: [WEB-1371](https://mobify.atlassian.net/browse/WEB-1371)
 **Linked PRs**: [generator-progressive-web#11](https://github.com/mobify/generator-progressive-web/pull/11), [platform-generater#42](https://github.com/mobify/platform-generator/pull/42)

## Changes
- Added `ui_kit_value` values on many properties that should be replaced with their UI Kit equivalents when the project is generated.

## Todos
- ~~(if applicable) analytics events instrumented to measure impact of change~~

## How to test-drive this PR
**With `generator-progressive-web`**
- See instructions in the [`generator-progressive-web` PR](https://github.com/mobify/generator-progressive-web/pull/11)

**With `platform-generator`**
- coming soon...

### CODE REVIEW

**Comment by jeremywiebe** [comment]
@jeffkamo Not sure how you're tracking the needed changes in the new Platform Generator.  Maybe a JIRA ticket linked to from here?

File: web/app/preloader/preload.css [hunk 1]
```diff
@@ -46,7 +46,7 @@ body::after {
     width: 100%;
     height: 52px; /* Same height as .t-header */
 
-    background: #41378c; /* Same as $brand-color */
+    background: #41378c; /* Same as $brand-color */  /* ui_kit_value: #005c83; /* Same as $secondary-brand-color */
```
Comment (jeremywiebe) [inline web/app/preloader/preload.css]: There's 3 comments here... is this right?  Or does the `ui_kit_value` not need to be first?

Comment (jeremywiebe) [inline web/app/preloader/preload.css]: Ooooooooooh, I get it.  The third comment is for the `ui_kit_value`?

File: web/app/styles/_variables.scss [hunk 2]
```diff
@@ -109,43 +109,43 @@ $bold-font-weight: 700;
...
+$dark-accent-color: #b21c30; // ui_kit_value: darken($accent-color, 15%);
 
 // Primary Action
 $primary-action-color: #dc0a3c; // red
 $light-primary-action-color: lighten($primary-action-color, 15%);
 $dark-primary-action-color: darken($primary-action-color, 15%);
 
 // Secondary Action
-$secondary-action-color: #ff852c; // orange
+$secondary-action-color: #ff852c; // ui_kit_value: #0288a7; // orange
```
Comment (jeremywiebe) [inline web/app/styles/_variables.scss]: In this case `// orange` is going to be preserved, but the `ui_kit_value` isn't orange, is it?

File: web/app/preloader/preload.css (hunk 1 above)
Comment (jeffkamo) [inline web/app/preloader/preload.css]: That is correct! The first comment describes the Merlin's Potions value, the last comment describes the UI Kit value.

File: web/app/styles/_variables.scss (hunk 2 above)
Comment (jeffkamo) [inline web/app/styles/_variables.scss]: You're right, will fix

**Comment by jeffkamo** [comment]
@jeremywiebe regarding the Jira ticket, it's already linked in the description :)

**Pull request merged**